moabb/datasets/physionet_mi.py

```diff
@@ -171,3 +176,41 @@ def _load_data(self, subject, runs, path=None, force_update=False, verbose=None)
             p = data_dl(url, sign, path, force_update, verbose)
             data_paths.append(p)
         return data_paths
+
+    def _create_stim_channels(self, raw):
```

Comment:
if you have created _create_stim_channels_with_selected_ids(), this function is not useful anymore, no ?

Reply:
Super Yes! @sebVelut. solved now!